extensions/adobe/experience/campaign/experienceevent-all.schema.json

```
@@ -126,6 +126,19 @@
               "title": "External ID",
               "type": "string",
               "description": "An ID generated by the parent activity invoking service to check later if the incoming experienceEvent was generated using this ID."
+            },
+            "xdm:offerProposed": {
```

If you use the mixin then you don't define a wrapping property. Instead it is added in two places:
- to the meta:extends` list
- to the schema validation rules at the bottom of the schema

line 15
```suggestion
            "meta:extends": [
                "https://ns.adobe.com/experience/decisioning/proposition#/definitions/propositions-mixin"
            ]
```

line 306
```suggestion
    {
      "$ref": "https://ns.adobe.com/experience/decisioning/proposition#/definitions/propositions-mixin"
    }
```

However for the offer clicked, which is not a proposition mixin it is correct to define a new property.

---

Sure